src/api/color.rs

```diff
@@ -44,6 +45,21 @@ pub enum ChromaSampling {
   Cs400,
 }
 
+impl fmt::Display for ChromaSampling {
```

Comment:
This should be a different PR (and could land immediately)

Reply:
I'm not sure there's a use in PR'ing this separately, because it doesn't do anything without being a part of this PR.